website/fiddle/package.json

```
@@ -22,15 +22,15 @@
     "dev": "node patch.js && node build_patch_ui5.js && webpack-dev-server  --content-base public"
   },
   "dependencies": {
-    "@luigi-project/client": "1.21.0",
-    "@luigi-project/core": "1.21.0",
+    "@luigi-project/client": "1.21.1-dev.202205122229",
+    "@luigi-project/core": "1.21.1-dev.202205122229",
```

```suggestion
    "@luigi-project/client": "1.22.0",
    "@luigi-project/core": "1.22.0",
```
We could apply to the latest version now :)

---

just did it :)